explainer.html

```diff
@@ -225,6 +225,29 @@ <h2>Defining an RDF Dataset Hash and/or Signing RDF Datasets</h2>
     </div>
   </section>
 
+  <section id="separation">
+    <h2>Separation of Concepts in the Deliverables</h2>
+    <p>
+      This section provides some background for the structure of <a href="./index.html#deliverables">deliverables</a> in the charter.
+    </p>
+
+    <p>
+      An attentive reader of the <a href="index.html">Linked Data Signatures Working Group Charter</a> may realize that two deliverables (i.e., the <a href="./index.html#hash">“RDF Dataset Hash”</a> and the <a href="./index.html#integrity">“Linked Data Integrity”</a> specifications) rely on the same <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs 1.0</a> draft Community Group Report. The main reason is a separation of concepts to make the deliverable structure more readable, but to also reflect different possible use cases. 
```

Comment:
Same question as above
```suggestion
      An attentive reader of the <a href="index.html">Linked Data Signatures Working Group Charter</a> may realize that two deliverables (i.e., the <a href="./index.html#hash">“RDF Dataset Hash”</a> and the <a href="./index.html#integrity">“Linked Data Integrity”</a> specifications) rely on the same <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs 1.0</a> draft Community Group Report. The main reason is a separation of concerns to make the deliverable structure more readable, but to also reflect different possible use cases. 
```

Reply:
same here...

But I can of course be convinced why "concern" would be better.

Reply:
No, if that's what you meant, I get it. It's just that "separation of conce*" triggers a pavlovian response from my brain. And it seemed to make sense as well... :)